modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/test/components/instance-list-page/InstanceListPageItem.es.js

```diff
@@ -412,7 +412,7 @@ describe('The instance list item should', () => {
 		expect(setInstanceId).toBeCalledWith(1);
 	});
 
-	it('set BulkReassign modal visualization by clicking the reassign task button', () => {
```

Comment:
I did an analysis of these tests but I can't trigger DropDown because it is disabled and this is related to the component's business rule, I think this test we can ping the team to fix because it's necessary to think about the test strategy here, it's strange test something the user cannot click 🤷‍♂️.